src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -24,6 +24,10 @@ provideEditingConfig(Homepage, {
     pisaUrl: {
       title: 'PisaSales Portal API URL',
     },
+    jwtPisaSalesApiUrl: {
+      title: 'JWT PisaSales API URL',
+      description: 'Without trailing slash or "portal" path.',
```

Comment:
* [ ] FYI A more helpful description would answer "What is this for?", maybe "Where can I look this up?".